website/formideploy.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  lander: {
+    name: "react-live",
+  },
+  build: {
+    // Build output directory.
+    // Docusaurus defaults to `build`.
+    // We use `docusaurus build --out-dir build/open-source/docusaurus` to build intermediate real dirs.
+    dir: "build",
```

Comment:
I think in package.json, for the build script, you'll want to update the output dir. And then also check the baseUrl in docusaurus config to make sure it matches. Groqd should have an example of those two things (reviewing from mobile right now)

Reply:
Thank you for catching that. Yeah, the paths were off. I believe I pushed the correct URLs now that mirror our configuration for `groqd`.